test/augmentation/test_augmentation.py

```diff
@@ -4187,3 +4189,55 @@ def test_feature_median_blur(self, device, dtype):
         )
 
         utils.assert_close(out, expected)
+
+
+class TestRandomDownScale:
+    @staticmethod
+    def __check_equal_with_accuracy(in_arr, out_arr, acc):
+        in_arr = np.array(in_arr)
+        out_arr = np.array(out_arr)
+        return all(np.abs(in_arr[-2:] - out_arr[-2:]) <= acc * out_arr[-2:]) and all(in_arr[:-2] == out_arr[:-2])
```

Comment:
It should be tested as tensors, not with numpy array. You can use kornia.testing.assert_close method or `self.assert_close` if you use `BaseTester`